tensorboard/plugins/profile/tf_op_profile/tf-op-table.html

```diff
@@ -62,33 +65,43 @@
 .entry.selected > .header{
   font-weight: bold;
 }
-.entry .header:hover {
+.entry .row:hover {
   background-color: rgba(0,0,0,0.05);
 }
-.entry > .header > .disclosure:before {
+.entry > .row > .disclosure:before {
   display: inline-block;
   width: 1.2em;
   content: ' ';
 }
-.entry.parent > .header > .disclosure:before {
+.entry.parent > .row > .disclosure:before {
   content: '\25b6';
 }
-.entry.parent.expanded > .header > .disclosure:before {
+.entry.parent.expanded > .row > .disclosure:before {
   content: '\25bc';
 }
-.entry.parent.expanded:not(.selected) > .header .name {
+.entry.parent.expanded:not(.selected) > .row .name {
   opacity: 0.6;
 }
 </style>
+<template is="dom-if" if="[[_eq(level, 0)]]">
+  <div class="row header">
+    <span class="name">Time</span>
+    <span class="disclosure" style="{{_paddingDisclosure(0)}}"></span>
+    <span class="name">Op Name</span>
+  </div>
+</template>
 <div class$="{{_classEntry(node, level)}}">
-  <div class="header"
+  <div class="row"
        on-click="_handleHeaderClick"
        on-mouseenter="_handleHeaderMouseEnter"
        on-mouseleave="_handleHeaderMouseLeave">
     <div class="bar" style$="{{_styleBar(node)}}"></div>
     <span class="time">{{_percent(node.metrics.time)}}</span>
     <span class="disclosure" style="{{_paddingDisclosure(level)}}"></span>
     <span class="name">{{node.name}}</span>
+    <template is="dom-if" if="[[_hasProvenance(node)]]">
```

Comment:
maybe simpler just to generate the span either way and leave it empty if there's no provenance info